stripe-patrons-data/src/main/scala/com/gu/patrons/conf/PatronsIdentityConfig.scala

```diff
@@ -8,7 +8,7 @@ case class PatronsIdentityConfig(apiUrl: String, apiClientToken: String)
 
 object PatronsIdentityConfig extends ConfigService {
   def fromParameterStoreSync(stage: Stage) = {
-    SafeLogger.info(s"Loading Identity config")
+    logger.info(s"Loading Identity config")
```

Comment:
now classes extend (directly or indirectly) `SafeLogging`, which has a logger with a reference to the class name.  Previously we called the object `SafeLogger` which had a logger that referenced its own name.

Overall that means we log with the correct class name, which makes the logs more useful.